scipy/stats/mstats_basic.py

```diff
@@ -1303,6 +1303,21 @@ def trima(a, limits=None, inclusive=(True,True)):
         Tuple of (lower flag, upper flag), indicating whether values exactly
         equal to the lower (upper) limit are allowed.
 
+    Examples
+    --------
+    >>> from scipy.stats.mstats import trima
+
+    >>> a = np.arange(10)
+
+    The interval is left-end-closed, i.e., `[2, 8)`. Any value falling into
```

Comment:
Thanks for adding this example, just a minor comment: can you reword slightly, i.e. "Trim the array by keeping only values in the interval …" or something like that. (I found the introduction "The interval is left-end closed" confusing.)